tests/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/Setup/PackageContentTests.cs

```diff
@@ -14,27 +15,34 @@ namespace Microsoft.VisualStudio.Setup
     [UsesVerify]
     public sealed class PackageContentTests
     {
+        // These files are only added as part of signing.
+        private const string DigitalSignature = "package/services/digital-signature";
+        private const string Rels = "_rels/.rels";
```

Comment:
With the use of MicroBuild signing, there are 3 new `digital-signature` files added to the NuGet packages. The `.rels` file gets added only for ProjectSystem/Editors during signing. It exists in Common because that uses a different packing process. We simply ignore these extra files for the purpose of this test.

- _rels/.rels
- package/services/digital-signature/_rels/origin.psdor.rels
- package/services/digital-signature/origin.psdor
- package/services/digital-signature/xml-signature/[RANDOM].psdsxs

There is a special case for the Common test as it ends with these generated files. Thus, the ending `,` logic changes. So, for non-signed builds, we need to replace the non-comma line with a comma line.